### DISCUSSION TITLE
04/069-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%B9%9D%E7%AB%A0-%E4%B8%A4%E9%87%8D%E4%B8%96%E7%95%8C

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#100] [General]
# 04/069-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%B9%9D%E7%AB%A0-%E4%B8%A4%E9%87%8D%E4%B8%96%E7%95%8C

在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。

https://tts.liuli.moe/04/069-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%B9%9D%E7%AB%A0-%E4%B8%A4%E9%87%8D%E4%B8%96%E7%95%8C.html

**Comment by Nyalice1** [comment]
千代梨香-前文中是梨花

**Reply by iolliotier** [reply]
这个是一个月前的老文本了，新PR里问题已解决
